app/src/home/index.tsx

```diff
@@ -42,10 +46,61 @@ const Boroughs = styled.div`
 `;
 
 interface Props {
-  user: Object,
+  user: User,
 }
 
 function Home(props: Props) {
+  const { user } = props;
+  const firebaseService = new FirebaseService();
+  const [availableTix, setAvailableTix] = useState<number>(0);
+  const [enteredRaffleTix, setEnteredRaffleTix] = useState<number>(0);
+  const [numCompletedMHTNTasks, setNumCompletedMHTNTasks] = useState<number>(0);
+  const [numCompletedBKLYNTasks, setNumCompletedBKLYNTasks] = useState<number>(0);
+  const [numCompletedQNSTasks, setNumCompletedQNSTasks] = useState<number>(0);
```

Comment:
This is totally fine and obviously works -- but curious if you weighed instead having one object state like:
```js
{
  manhattan: 0,
  brooklyn: 0,
  queens: 0
}
```

eventually we should probably refactor the boroughs to an enum and we could set them as the keys too.

Anyway, just a suggestion!

Reply:
Definitely think this is cleaner/more readable. I just quickly refactored this to have this object structure but ran into an infinite looping issue lol. 

I can definitely revisit this, I like the idea of using enums and keys!

Reply:
Hey, sent you a message! Definitely more elegant but in the essence of time, merging this in now and we can refactor later!